samples/tfm_integration/CMakeLists.inc

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2019 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+project(NONE)
+
+# Make sure we are using a compatible board target.
+if((${BOARD} STREQUAL "mps2_an521_nonsecure"))
```

Comment:
We shouldn't need BOARD stuff here, it should move into the sample/test app files.